cafs/mapstore.go

```diff
@@ -96,16 +96,27 @@ func (m *MapStore) AddConnection(other *MapStore) {
 func (m MapStore) Print() (string, error) {
 	buf := &bytes.Buffer{}
 	for key, file := range m.Files {
-		data, err := ioutil.ReadAll(file.File())
-		if err != nil {
-			return "", err
+		f := file.File()
+		if !f.IsDirectory() {
+			data, err := ioutil.ReadAll(f)
+			if err != nil {
+				return "", err
+			}
+			fmt.Fprintf(buf, "%s:%s\n\t%s\n", key, f.FullPath(), string(data))
 		}
-		fmt.Fprintf(buf, "%s:%s\n\t%s\n", key, file.File().FileName(), string(data))
 	}
 
 	return buf.String(), nil
 }
 
+// ObjectCount returns the number of content-addressed objects in the store
+func (m MapStore) ObjectCount() (objects int) {
+	for range m.Files {
```

Comment:
pretty sure this is just code that's been refactored into nothing. This entire `cafs` package will be deleted in a follow up PR, but I'll make the change anyway because "pride".